views/building-hours/format-building-hours.js

```diff
@@ -14,14 +14,18 @@ export function formatBuildingHours(hoursInfo: BuildingInfoType, now: momentT):
 
   let {open, close} = hours
   let openStatus = isBuildingOpen(hoursInfo, now)
+  let openString = open.format(RESULT_FORMAT)
+  let closeString = close.format(RESULT_FORMAT)
 
   if (openStatus === 'Almost Closed') {
     let closingIn = close.diff(now, 'minutes') + 1
 
     return `Closing in ${closingIn} minutes`
+  } else if (openStatus === 'Almost Open') {
+    let openingIn = open.diff(now, 'minutes') + 1
+
+    return `${openString} / ${closeString} – Opening in ${openingIn} minutes`
```

Comment:
I didn't catch this on the last PR, but I will mention it here. Will this end up saying "1 minutes" at some point?